mdhhockey/constants.py

```diff
@@ -1,12 +1,37 @@
+import configparser
+
+# Config File (for vars that shouldn't be uploaded to GitHub)
+config = configparser.ConfigParser()
+config.read(['/home/jeremy/mdh-hockey/mdhhockey/config.cfg'])
+
+# Root of the project to ensure absolute paths
+PROJECT_ROOT = config['project']['root']
+
 # Directories to cache GET requests in
-CACHE_DIR = 'response_cache'
+CACHE_DIR = f'{PROJECT_ROOT}/response_cache'
 PLAYERS_CACHE_SUBDIR = 'players'
 
-INPUTS_DIR = 'inputs'
+# Azure/OneDrive variables
+AZURE_CLIENT_ID = config['azure']['client_id']
+AZURE_SCOPES = ['https://graph.microsoft.com/.default']
+AZURE_AUTHORITY = 'https://login.microsoftonline.com/consumers'
+AZURE_TOKEN_CACHE = f'{CACHE_DIR}/cache.bin'
+AZURE_USER = config['azure']['user']
+CAPFRIENDLY_GRAPH_URL_ROOT = "https://graph.microsoft.com/v1.0/me/drive/root:/Hockey/MDH/MDH-CapFriendly.xlsx:/workbook/"
```

Comment:
```suggestion
CAPFRIENDLY_GRAPH_URL_ROOT = "https://graph.microsoft.com/v1.0/drives/56555516577eabf8/items/56555516577EABF8!64168/workbook"
```

Based off our DMs in Discord - this is the URL that worked for me. Does it work for you as well?

Reply:
Note that this is a public Github repository, so not sure if these values should be private or not.

Reply:
Should be fine -- you'd need share permissions on the drive/file anyway to do anything.

Reply:
Cool, I committed this suggestion then.